src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
```

Comment:
The 'body' selector is a tag name, which goes against the checklist. It might be better to use a class to style the body. Also, there is no fallback font specified for Roboto. If the Roboto font fails to load, the browser will use a default font, which might not look as intended. It's a good practice to specify a fallback font.